EIPS/eip-6800.md

```diff
@@ -42,7 +42,7 @@ def group_to_field(point: Point) -> int:
     if point == bandersnatch.Z:
         return 0
     else:
-        return int.from_bytes(point.serialize(), 'little') % BANDERSNATCH_MODULUS
+        return int.from_bytes(point.map_to_base_field().to_bytes(32, 'little'), 'little') % BANDERSNATCH_MODULUS
```

Comment:
This was incorrect, since we don't serialize the point to map a point to a scalar-field element. (i.e: we do `X/Y`  (affine form)).

To avoid messing with coordinate systems, I "hand wave" it with a `map_to_base_field()` call, which does what it says: maps the point to a base-field element (i.e: `X/Y`). Then we do the transformation of that to bytes, and re-interpret as a scalar-field.

All the above can be also be summarized in code as shown in [go-ipa here](https://github.com/crate-crypto/go-ipa/blob/master/banderwagon/element.go#L259-L264).

Reply:
yeah ok we can also add `map_to_field` to the EIP for now (as python code) and when it's specified (in a specific repo or an EIP) we can remove it.